### PR TITLE
Corrected the twitter share  button README_EN.md.Solves the issue #293

### PR DESCRIPTION
**Pull request by PAVAN507-STAR**
Corrected twitter share button in readme_en.md by comparing it with readme.md.Closes the issue #293

### CODE REVIEW

File: docs/README_EN.md [hunk 1]
```diff
@@ -390,4 +390,4 @@ This project is licensed under the [MIT LICENSE](/LICENSE).
 
 This project is inspired by great open source projects designed for beginners, such as [first-contributions](https://github.com/firstcontributions/first-contributions) and [Contribute-To-This-Project](https://github.com/Syknapse/Contribute-To-This-Project).
 
-[twttr-share]: https://twitter.com/intent/tweet?text=Start your OSS activities with a tutorial in full Japanese🚀&url=https://github.com/first-contributions-ja/first-contributions-ja.github.io&hashtags=Programming 'Tweet this project'
\ No newline at end of file
+[twttr-share]: https://twitter.com/intent/tweet?text=Start your OSS activities with a tutorial in full Japanese🚀&url=https://github.com/first-contributions-ja/first-contributions-ja.github.io&hashtags=Programming 'Tweet this project'
```
Comment (kazzyfrog) [inline docs/README_EN.md:393]: Spaces in 
`Start your OSS activities with a tutorial in full Japanese🚀`
should be replaced with `%20`.

This allows it to be treated as a single URL string.

File: docs/README_EN.md [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <img src="https://img.shields.io/badge/first_contributions-%F0%9F%94%B0-white?style=flat" alt="Static Badge">
 </p>
 
-[![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)][twttr-share]
+![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)
```
Comment (kazzyfrog) [inline docs/README_EN.md:13]: This code, in its previous state, is fine.

As it is now, the image is displayed and we need to treat it as a Link.
[ref](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links)

**Review by kazzyfrog — COMMENTED** [review]
Thanks for the PR 🙌
I found two points that would make this change better.
Please check them!

**Comment by PAVAN507-STAR** [comment]
@kazzyfrog idk how this got closed.Can i rasie a pr again?

**Comment by kazzyfrog** [comment]
@PAVAN507-STAR 
Did you delete the forked repository by any chance?
If you deleted the forked repository before the changes were merged, the PR will be closed.

PR will incorporate changes in the fork repository, so if it is deleted, it will no longer work properly.

**Comment by kazzyfrog** [comment]
Yes please!

Thank you for your contribution!
If you have any other needs, we can support you👍

**Comment by PAVAN507-STAR** [comment]
@kazzyfrog if u don't mind can u share what changes need to be done once again ,the readme seems to be changed

**Comment by kazzyfrog** [comment]
Sure!

Please replace the half-width spaces in line 393, 
specifically in the text `Start your OSS activity with a tutorial🚀`, with `%20`.
 This will allow the text to be treated as a single URL string.

This allows the user to be redirected to X site once the X-share button is pressed!

**Pull request closed without merging**